src/runtime_src/driver/xclng/drm/xocl/devices.h

```diff
@@ -939,7 +941,8 @@ enum subdev_id {
 	{ XOCL_PCI_DEVID(0x10EE, 0x5001, PCI_ANY_ID, USER_DSA52) },	\
 	{ XOCL_PCI_DEVID(0x10EE, 0x5005, PCI_ANY_ID, USER_DSA52) },	\
 	{ XOCL_PCI_DEVID(0x10EE, 0x5009, PCI_ANY_ID, USER_DSA52) },	\
-	{ XOCL_PCI_DEVID(0x13FE, 0x0065, PCI_ANY_ID, USER_XDMA) },	\
+	{ XOCL_PCI_DEVID(0x13FE, 0x0065, PCI_ANY_ID, USER_DSA52) },	\
```

Comment:
Changing USER_XDMA to USER_DSA52 may introduce compatible issue with old DSA (13fe 6a). Could you confirm that you have tested your change with old DSA 13fe,65?